src/systems/fem_context.C

```diff
@@ -250,8 +259,16 @@ void FEMContext::some_interior_gradient(unsigned int var, unsigned int qp, Outpu
 
 #ifdef LIBMESH_ENABLE_SECOND_DERIVATIVES
 template<typename OutputType,
+         void (FEMContext::*fe_getter)
+           (unsigned int,
+            FEGenericBase
+              <typename TensorTools::MakeReal
+                <typename TensorTools::DecrementRank
+                  <typename TensorTools::DecrementRank
+                    <OutputType>::type>::type>::type> *&)
```

Comment:
It's possible with a C++11 using statement, but not in C++03.  OutputType is a free argument and appears as a template argument to DecrementRank here, and "template typedef" doesn't exist.  Best we could do is to add a nested templated class to the header and use three typedefs inside that class.  I'll bet that ends up being just as many lines of code, but I'll try it out if you like.